### PR TITLE
upgrade airnow2ioda converter

### PR DESCRIPTION
**Pull request by ytangnoaa**
## Description

*(Instructions: replace text in this and all sections with your own text)*

Provide a detailed description of what this PR does.

remove the duplicated observation records
change variable name of pm25_tot to pm25
add real station_elevation (meter) in place (the existing one uses dummy value)

What problem does it fix? What new capability does it add?
remove the duplicated record,  add the real station_elevation. The variable name change is for the new AIRNow UFO 
https://github.com/JCSDA-internal/ufo/pull/1666

[See the JEDI Documentation](https://jointcenterforsatellitedataassimilation-jedi-docs.readthedocs-hosted.com/en/latest/inside/practices/pullrequest.html) for further information.

### Issue(s) addressed

Link the issues to be closed with this PR
N/A
If this does not close an existing issue, then [be sure to add a Pipeline, Label, Estimate, Assignees, and Epic](https://jointcenterforsatellitedataassimilation-jedi-docs.readthedocs-hosted.com/en/latest/inside/practices/issues.html)

## Acceptance Criteria (Definition of Done)

What does it mean for this PR to be finished?
It can pass the regression test and convert the data to the desired format

## Dependencies
N/A

If there are PRs that need to be merged before or along with this one, please add "waiting for another PR" label and list the dependencies in the other repositories (example below). Note that the branches in the other repositories should have matching names for the automated tests to pass.
Waiting on the following PRs:
- [ ] waiting on JCSDA/eckit/pull/<pr_number>
- [ ] waiting on JCSDA/atlas/pull/<pr_number>

## Impact

Please list the other repositories (if any) that this PR will require changes in (example below)
This PR is the upstream change for implementing https://github.com/JCSDA-internal/ufo/pull/1666
- [ ] saber
- [ ] ufo
- [ ] ...

### CODE REVIEW

File: src/chem/airnow2ioda-nc.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 from datetime import datetime
 from pathlib import Path
 
-IODA_CONV_PATH = Path(__file__).parent/"@SCRIPT_LIB_PATH@"
+IODA_CONV_PATH = Path(__file__).parent/"../lib/pyiodaconv"
```
Comment (CoryMartin-NOAA) [inline src/chem/airnow2ioda-nc.py:10]: Don't change this.

File: src/chem/airnow2ioda-nc.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
```
Comment (CoryMartin-NOAA) [inline src/chem/airnow2ioda-nc.py:1]: I think this is supposed to stay at python3 for compatibility issues on certain platforms.

File: src/chem/airnow2ioda-nc.py [hunk 3]
```diff
@@ -127,31 +127,31 @@ def add_data(infile, sitefile):
...
     loc_mdata['station_id'] = writer.FillNcVector(c, 'string')
 
-    outdata[varDict['pm25_tot']['valKey']] = np.array(f3['PM2.5'].fillna(nc.default_fillvals['f4']))
+    outdata[varDict['pm25']['valKey']] = np.array(f3['PM2.5'].fillna(nc.default_fillvals['f4']))
     outdata[varDict['o3']['valKey']] = np.array((f3['OZONE']/1000).fillna(nc.default_fillvals['f4']))
-    for i in ['pm25_tot', 'o3']:
+    for i in ['pm25', 'o3']:
         outdata[varDict[i]['errKey']] = np.full((nlocs), 0.1)
-        outdata[varDict[i]['qcKey']] = np.full((nlocs), 0)
+        outdata[varDict[i]['qcKey']] = np.full((nlocs), 1)
```
Comment (CoryMartin-NOAA) [inline src/chem/airnow2ioda-nc.py:154]: Generally, a QC value of 1 mean that it is a 'bad' observation...

**Review by CoryMartin-NOAA — CHANGES_REQUESTED** [review]
Some changes at the top of the script that need to be revoked.
Unrelated to this PR, but this converter needs upgraded to use IODAv2 and the IODA python API.

File: src/chem/airnow2ioda-nc.py (hunk 1 above)
Comment (ytangnoaa) [reply src/chem/airnow2ioda-nc.py]: OK, I updated the branch

File: src/chem/airnow2ioda-nc.py (hunk 3 above)
Comment (ytangnoaa) [reply src/chem/airnow2ioda-nc.py]: Thank you. I changed back to 0

**Comment by CoryMartin-NOAA** [comment]
Coding norms test fails:
```
checking ./aeronet_aaod2ioda.py
checking ./aeronet_aod2ioda.py
checking ./airnow2ioda-nc.py
./airnow2ioda-nc.py:111:76: E231 missing whitespace after ','
./airnow2ioda-nc.py:111:84: E231 missing whitespace after ','
./airnow2ioda-nc.py:111:93: E231 missing whitespace after ','
./airnow2ioda-nc.py:111:104: E231 missing whitespace after ','
./airnow2ioda-nc.py:115:1: W293 blank line contains whitespace
checking ./modis_aod2ioda.py
checking ./tropomi_no2_nc2ioda.py
checking ./viirs_aod2ioda.py
4       E231 missing whitespace after ','
1       W293 blank line contains whitespace
```

**Review by CoryMartin-NOAA — APPROVED** [review]

**Review by jeromebarre — APPROVED** [review]
PR looks good. Maybe write an issue for including potential useful other species in the future such as CO, NO2, PM10 and SO2. I don't think you should wait for Mark review as he is not working at JCSDA anymore. So he can be removed from the reviewer list.

**Pull request merged**